src/quo/animated.cljs

```diff
@@ -1,12 +1,14 @@
 (ns quo.animated
   (:refer-clojure :exclude [set delay divide])
-  (:require [reagent.core :as reagent]
-            [quo.react-native :as rn]
-            [quo.gesture-handler :as gh]
-            [oops.core :refer [oget ocall]]
-            ["react-native-reanimated" :default animated :refer (clockRunning EasingNode)]
-            ["react-native-redash/lib/module/v1" :as redash]
-            quo.react)
+  (:require
+   [reagent.core        :as reagent]
+   [quo.react-native    :as rn]
+   [quo.gesture-handler :as gh]
+   [oops.core           :refer [oget ocall]]
+   ["react-native-reanimated" :default animated
+                              :refer   (clockRunning EasingNode)]
+   ["react-native-redash/lib/module/v1" :as redash]
+   quo.react)
```

Comment:
a bit weird how `:refer/:default/:as` are aligned in last three lines

Reply:
Fixed by set the `:max-variance` to 1000